sdk/scheduler/src/test/java/com/mesosphere/sdk/config/validate/PodSpecsCannotChangeNetworkRegimeTest.java

```diff
@@ -0,0 +1,134 @@
+ */
+public class PodSpecsCannotChangeNetworkRegimeTest {
+    private static final ConfigValidator<ServiceSpec> VALIDATOR = new PodSpecsCannotChangeNetworkRegime();
+
+    @Mock
+    private PodSpec overlaypod1;
+    @Mock
+    private PodSpec overlaypod2;
+
+    @Mock
+    private PodSpec bridgepod1;
+    @Mock
+    private PodSpec bridgepod2;
+
+    @Mock
+    private PodSpec hostpod1;
+    @Mock
+    private PodSpec hostpod2;
+
+    //private List<PodSpec> overlayPods = new ArrayList<>(Arrays.asList(overlaypod1, overlaypod2));
+    //private List<PodSpec> hostPods = new ArrayList<>(Arrays.asList(hostpod1, hostpod2));
+    //private List<PodSpec> bridgePods = new ArrayList<>(Arrays.asList(bridgepod1, bridgepod2));
+    private List<PodSpec> overlayPods = new ArrayList<>();
+    private List<PodSpec> hostPods = new ArrayList<>();
+    private List<PodSpec> bridgePods = new ArrayList<>();
+
+    @Before
+    public void beforeEach() {
+        MockitoAnnotations.initMocks(this);
+
+        DefaultNetworkSpec overlayNetworkSpec, bridgeOverlayNetworkSpec;
+        overlayNetworkSpec = new DefaultNetworkSpec(DcosConstants.DEFAULT_OVERLAY_NETWORK,
+                Collections.emptyMap(), Collections.emptyMap());
+        bridgeOverlayNetworkSpec = new DefaultNetworkSpec("mesos-bridge",
+                Collections.emptyMap(), Collections.emptyMap());
+
+        List<NetworkSpec> overlayNetworkSpecs = new ArrayList<>(Collections.singletonList(overlayNetworkSpec));
+        List<NetworkSpec> bridgeNetworkSpecs = new ArrayList<>(Collections.singletonList(bridgeOverlayNetworkSpec));
+        List<NetworkSpec> onHostNetwork = new ArrayList<>();
+
+
+        when(overlaypod1.getType()).thenReturn(TestConstants.POD_TYPE + "-1");
+        when(overlaypod2.getType()).thenReturn(TestConstants.POD_TYPE + "-2");
+        when(overlaypod1.getNetworks()).thenReturn(overlayNetworkSpecs);
+        when(overlaypod2.getNetworks()).thenReturn(overlayNetworkSpecs);
+        overlayPods.addAll(Arrays.asList(overlaypod1, overlaypod2));
+
+        when(bridgepod1.getType()).thenReturn(TestConstants.POD_TYPE + "-1");
+        when(bridgepod2.getType()).thenReturn(TestConstants.POD_TYPE + "-2");
+        when(bridgepod1.getNetworks()).thenReturn(bridgeNetworkSpecs);
+        when(bridgepod2.getNetworks()).thenReturn(bridgeNetworkSpecs);
+        bridgePods.addAll(Arrays.asList(bridgepod1, bridgepod2));
+
+        when(hostpod1.getType()).thenReturn(TestConstants.POD_TYPE + "-1");
+        when(hostpod2.getType()).thenReturn(TestConstants.POD_TYPE + "-2");
+        when(hostpod1.getNetworks()).thenReturn(onHostNetwork);
+        when(hostpod2.getNetworks()).thenReturn(onHostNetwork);
+        hostPods.addAll(Arrays.asList(hostpod1, hostpod2));
+
+
+    }
+
+    private void testConfigTransition(List<PodSpec> oldPodSpecs, List<PodSpec> newPodSpecs,
+                                 int expectedHomoTransitionErrors, int expectedHeteroTransitionErrors)
```

Comment:
nitpick: `expectedSameNetworkErrors` and `expectedChangedNetworkErrors` makes more obvious sense to me.